JSQMessagesViewController/Views/JSQMessagesCollectionView.h

```diff
@@ -73,6 +73,26 @@
 @property (strong, nonatomic) UIColor *typingIndicatorEllipsisColor;
 
 /**
+ *  The image to be used as an avatar on a user's typing indicator. If not set, no avatar image will be shown. Setting this image will hide the typing indicator message bubble and ellipsis image.
+ */
+@property (strong, nonatomic) UIImage *typingIndicatorAvatarImage;
+
+/**
+ *  The string to indicate that a user is typing. If not set, a generic message will be shown. Setting this string will hide the typing indicator message bubble and ellipsis image.
+ */
+@property (copy, nonatomic) NSString *typingIndicatorMessage;
```

Comment:
oh wait. we don't want `message` and `avatarImage` to be properties.

these need to be dynamic, otherwise it doesn't support group chat.